### PR TITLE
User can create a playlist.

### PR DESCRIPTION
**Pull request by NicBonetto**
screen shot 2017-06-19 at 9 29 11 am

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -168,3 +170,45 @@ $form.addEventListener('submit', function () {
...
+  addPlaylist(playlist, song)
+  var row = renderPlaylist(playlist, playIndex)
+
+  var $tbody = document.querySelector('tbody')
+
+  $tbody.appendChild(row)
+  playIndex++
+})
+
+$carousel.addEventListener('mouseover', function () {
```
Comment (thebearingedge) [inline main.js]: You you use CSS for this.

```css
.carousel {
  cursor: pointer;
}
```

**Review by thebearingedge — COMMENTED** [review]
Good job. I have a small CSS note. Also, I recommend adding some kind of "Add" button to your carousel so that the user knows they can add the song. Otherwise, I got the following error when running your page.
screenshot 2017-06-19 09 38 34

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Good job. I have a small CSS note. Also, I recommend adding some kind of "Add" button to your carousel so that the user knows they can add the song. Otherwise, I got the following error when running your page.
screenshot 2017-06-19 09 38 34

**Comment by NicBonetto** [comment]
screen shot 2017-06-19 at 3 08 26 pm

**Pull request merged**